client/structs/host_volumes.go

```diff
@@ -40,15 +41,41 @@ type ClientHostVolumeCreateResponse struct {
 	VolumeName string
 	VolumeID   string
 
-	// Path is the host path where the volume's mount point was created. We send
-	// this back to the server to make debugging easier.
+	// HostPath is the host path where the volume's mount point was created. We
+	// send this back to the server to make debugging easier.
 	HostPath string
 
 	// Capacity is the size in bytes that was actually provisioned by the host
 	// volume plugin.
 	CapacityBytes int64
 }
 
+type ClientHostVolumeRegisterRequest struct {
+	// ID is a UUID-like string generated by the server.
+	ID string
+
+	// Name is the name that group.volume will use to identify the volume
+	// source. Not expected to be unique.
```

Comment:
```suggestion
	// source. Not expected to be unique cluster-wide, but must be unique per node.
```

(not sure if that overflows your desired line length)